sqlmesh/utils/__init__.py

```diff
@@ -162,6 +162,19 @@ def nullsafe_join(join_char: str, *args: t.Optional[str]) -> str:
     return join_char.join(filter(None, args))
 
 
+def str_to_bool(s: t.Optional[str] = None) -> bool:
```

Comment:
Can we also use this here: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/__init__.py#L116